### DISCUSSION TITLE
getting unfiltered results of current analysis

### DISCUSSION THREAD

**Discussion by krummrey** [mcguirepr89/BirdNET-Pi#920] [Q&A]
Hi,
I like to see what results don't quite make it over the threshold. I can see the raw results in the log window. I'd love to print them out in Kiosk mode or some other view. Is there any way to get them into a PHP variable so that I can start tinkering with some sort of output?
I'm not a coder but have tweaked scripts here and there. Would love to get a starting point.

**Comment by ehpersonal38** [comment]
Haven't tested this, but I think if you can see this information if you print out `p_sorted` in in this line of code: https://github.com/mcguirepr89/BirdNET-Pi/blob/main/scripts/server.py#L251

**Reply by krummrey** [reply]
Thanks, looks like a good lead. Will give it a shot.
